sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -124,19 +133,21 @@
     private final ClientLogger logger = new ClientLogger(ServiceBusAdministrationAsyncClient.class);
     private final ServiceBusManagementSerializer serializer;
     private final RulesImpl rulesClient;
+    private final TokenCredential tokenCredential;
 
     /**
      * Creates a new instance with the given management client and serializer.
-     *
-     * @param managementClient Client to make management calls.
+     *  @param managementClient Client to make management calls.
      * @param serializer Serializer to deserialize ATOM XML responses.
+     * @param credential Credential to get additional tokens if necessary
      */
     ServiceBusAdministrationAsyncClient(ServiceBusManagementClientImpl managementClient,
-        ServiceBusManagementSerializer serializer) {
+                                        ServiceBusManagementSerializer serializer, TokenCredential credential) {
```

Comment:
I think this indentation no longer matches the other methods in this source file

Reply:
Yeah, we didn't align parameter methods. And there is a new line between the javadoc summary and params.

Reply:
For consistency, we should do an Objects.notRequired and update the javadoc for throws

Reply:
Added the additional check for consistency and will go back to the old indenation